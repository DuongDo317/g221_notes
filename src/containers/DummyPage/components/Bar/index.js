import React from 'react'
import styled from 'styled-components'
import Button from '@material-ui/core/Button'

const StyledButton = styled(Button)`
  background: linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%);
  border: 0;
  border-radius: 3px;
  box-shadow: 0 3px 5px 2px rgba(255, 105, 135, .3);
  color: white;
  height: 48px;
`

export default function Bar({ children }) {
  return <StyledButton>{children}</StyledButton>
}
