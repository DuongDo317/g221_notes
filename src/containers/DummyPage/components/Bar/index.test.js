import React from 'react'
import { render } from '@testing-library/react'

import Bar from './index'

describe('<Bar>', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<Bar>text</Bar>)
    expect(container.firstChild).toMatchSnapshot()
  })
})
