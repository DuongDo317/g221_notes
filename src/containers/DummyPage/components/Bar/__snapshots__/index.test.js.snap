// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bar> should render and match the snapshot 1`] = `
<button
  class="MuiButtonBase-root MuiButton-root sc-bdVaJa dYArxF MuiButton-text"
  tabindex="0"
  type="button"
>
  <span
    class="MuiButton-label"
  >
    text
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;
